Modules/Loadable/Segmentations/Widgets/qMRMLSegmentEditorWidget.h

```diff
@@ -276,6 +276,9 @@ public slots:
   /// Enable/disable masking using master volume intensity
   void toggleMasterVolumeIntensityMaskEnabled();
 
+  /// Toggle style of effect buttons to see button labels if needed.
```

Comment:
Hardcoding a specific label style would not be very bad, but it would be nicer if we remembered what was the label style and toggle between that style and no label.